tests/conftest.py

```diff
@@ -0,0 +1,14 @@
+import os
+import pytest
+
+
+INCLUDE_SKIPS = os.getenv("ROCM_TEST_INCLUDE_SKIPS", default=False)
+
+@pytest.hookimpl(optionalhook=True)
+def pytest_json_modifyreport(json_report):
+    """Get rid of skipped tests in reporting. We only care about xfails."""
+    if (not INCLUDE_SKIPS
+            and "summary" in json_report
+            and "total" in json_report["summary"]):
+        json_report["summary"]["unskipped_total"] = json_report["summary"]["total"] - json_report["summary"].get("skipped")
```

Comment:
```suggestion
        json_report["summary"]["unskipped_total"] = json_report["summary"]["total"] - json_report["summary"].get("skipped", 0)
```
You probably want .get() to default to 0 if it can't find skipped, otherwise it will return None by default.

Reply:
Yep. Definitely want that. Good idea.